api/http/handler/stacks/stack_delete.go

```diff
@@ -245,7 +245,16 @@ func (handler *Handler) deleteStack(userID portainer.UserID, stack *portainer.St
 		}
 
 		out, err := handler.KubernetesDeployer.Remove(userID, endpoint, manifestFiles, stack.Namespace)
-
+		if err != nil {
+			for _, manifest := range manifestFiles {
+				if exists, err := filesystem.FileExists(manifest); err != nil || !exists {
```

Comment:
to prevent overriding of the original error (L248), rename this variable
```suggestion
				if exists, fileExistsErr := filesystem.FileExists(manifest); fileExistsErr != nil || !exists {
```